src/meta.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     livescript: ">=1.0.0 <2.0.0",
     svelte: ">=3.0.0 <4.0.0",
     swc: ">=1.0.0 <2.0.0",
-    typescript: ">=2.0.0 <5.0.0",
+    typescript: ">=2.0.0 <6.0.0",
```

Comment:
`src/meta.js` is generated with a script in `tools` (from `package.json`; meta.js contains some other information from the package manifest as well), so these changes will be overwritten unless they're added to `package.json` as well.

I've added a commit to the PR to that effect.